### ISSUE TITLE
Missing pipeline components in quickstart widget 

### ISSUE THREAD

**Issue by justus-saul** [explosion/spaCy#8269]
## Which page or section is this issue related to?
https://spacy.io/usage/training#quickstart

## Issue
In the process of migrating to spacy 3 I wanted to retrain our models. I used the quickstart widget to configure everything to my liking and it was working really fine. So I thought.
A lot of our tests failed with the new model, because the lemmas were missing. Took me a bit but then I saw your "new 3.0" here https://spacy.io/api/lemmatizer

I then was able to add the lemmatizer to our config and everything is fine now, but I think that the  lemmatizer should be added to the quickstart widget as well.
Basically all pipeline components should be selectable there. If this not an oversight and a decision to not have it there, please at least add an info badge. This will save a lot of people from getting a headache.
